modules/rostests/winetests/msvcrt/misc.c

```diff
@@ -126,63 +129,62 @@ typedef struct _I10_OUTPUT_test {
 
 static const I10_OUTPUT_test I10_OUTPUT_tests[] = {
     /* arg3 = 0 */
-    { 0.0, 10, 0, {0, ' ', 1, "0"}, 1, "" },
-    { 1.0, 10, 0, {1, ' ', 1, "1"}, 1, "000000009" },
-    { -1.0, 10, 0, {1, '-', 1, "1"}, 1, "000000009" },
-    { 1.23, 10, 0, {1, ' ', 3, "123"}, 1, "0000009" },
-    { 1e13, 10, 0, {14, ' ', 1, "1"}, 1, "000000009" },
-    { 1e30, 30, 0, {31, ' ', 21, "100000000000000001988"}, 1, "" },
-    { 1e-13, 10, 0, {-12, ' ', 1, "1"}, 1, "000000000" },
-    { 0.25, 10, 0, {0, ' ', 2, "25"}, 1, "00000000" },
-    { 1.0000001, 10, 0, {1, ' ', 8, "10000001"}, 1, "00" },
+    { { 0x00000000, 0x00000000, 0x0000 /* 0.0 */ }, 10, 0, {0, ' ', 1, "0"}, 1, "" },
```

Comment:
Is this basically a fork of the corresponding wine test that is being done here?

Reply:
It is the test from current wine head, so yes, kindof a fork. I could sync the entire test, if that is preferred.

Reply:
Fine if it's closer to what Wine nowadays have.